### DISCUSSION TITLE
Blue Screen Issue After Setting Up KDE Plasma Desktop and Connecting via RDP

### DISCUSSION THREAD

**Discussion by xyf2002** [sileshn/ManjaroWSL2#19] [General]
### Requirements

- [X] Yes
- [ ] No

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Version

10.0.22621.3155

### Kernel Version

5.15.133.1-1

### Repro Steps

1. Install KDE Plasma and necessary packages with:
sudo pacman -S plasma kio-extras
sudo systemctl enable sddm.service --force
pacman -S xorg xorg-server xrdp
yay -S xorgxrdp
2. Enable and start `xrdp` service:
3. sudo systemctl enable xrdp.service
sudo systemctl start xrdp.service
sudo systemctl reboot
3. Add the specified code to `~/.xinitrc`.
4. Obtain IP address with `ip addr | grep eth0`.
5. Connect to the RDP server using `mstsc` with `ip:13389` and select the `xvnc` option.

### Expected Behavior

I expected to see the KDE Plasma desktop environment after successfully logging in through the RDP session.

### Actual Behavior

After entering my username and password, I am presented with a blue screen, and the desktop environment does not load.
![image](https://github.com/sileshn/ManjaroWSL2/assets/49825906/322857a6-8d7c-465e-975d-d51886bc6e7d)

**Comment by xyf2002** [comment]
![image](https://github.com/sileshn/ManjaroWSL2/assets/49825906/ba04edda-688d-4ae9-954f-b3ad440a16ed)

**Comment by sileshn** [comment]
I don't use a gui with wsl. Someone has already installed a kde desktop and provided instructions which is available in the readme file. 

On first glance, in step 6, you have tried to connect to rdp using ip:13389. I think its ip:3389

**Reply by xyf2002** [reply]
I changed it to 13389

**Reply by xyf2002** [reply]
I  changed the rdp ip to 13389 to avoid conflict, and my vnc error problem still has not been solved.
